docs/proposals/security-context-constraints.md

```diff
@@ -0,0 +1,284 @@
+## Abstract
+
+Security context constraints relate [security contexts](./security_contexts.md) and
+[service accounts](http://docs.k8s.io/design/service_accounts.md).  They are configured for a
+cluster, namespaces, and users within the cluster to govern how a [security context](./security_contexts.md) is established.
+
+## Motivation
+
+Administration of a multi-tenant cluster requires the ability to provide varying sets of permissions
+among the tenants, the infrastructure components, and end users of the system who may themselves be
+administrators within their own isolated namespace.
+
+Actors in a cluster may include infrastructure that is managed by administrators, infrastructure
+that is exposed to end users (builds, deployments), the isolated end user namespaces in the cluster, and
+the individual users inside those namespaces.  Infrastructure components that operate on behalf of a
+user (builds, deployments) should be allowed to run at an elevated level of permissions without
+granting the user themselves an elevated set of permissions.
+
+### Goals
```

Comment:
I think you accumulated an additional `#` here, and it skewed everything else from this point down in the doc.